controllers/teams/updater.js

```diff
@@ -1,7 +1,7 @@
 const { Team } = require('../../databases/db');
 const { imgController, fileController } = require('../filesControllers');
 const path = require('path');
-
+const responses= require('../../utils/responses/responses');
```

Comment:
```suggestion
const responses = require('../../utils/responses/responses');
```
Por favor, separa el = 😡